src/scala/com/github/johnynek/bazel_deps/IO.scala

```diff
@@ -115,83 +111,194 @@ object IO {
         System.exit(0)
     }
 
+  def orConst[A](optIO: Option[Result[A]], a: A): Result[A] =
+    optIO match {
+      case Some(io) => io
+      case None => IO.const(a)
+    }
+
+  def orUnit(optIO: Option[Result[Unit]]): Result[Unit] =
+    orConst(optIO, ())
```

Comment:
Is `orConst` used anywhere else? Wondering if we even need the separate method if we're only ever using `orUnit`